src/components/app/data/services/enterpriseCustomerUser.js

```diff
@@ -59,61 +65,75 @@ export async function fetchEnterpriseLearnerData(username, enterpriseSlug, optio
     page: 1,
   });
   const url = `${enterpriseLearnerUrl}?${queryParams.toString()}`;
-  const {
-    results: enterpriseCustomersUsers,
-    response: enterpriseCustomerUsersResponse,
-  } = await fetchPaginatedData(url);
-  const { enterpriseFeatures } = enterpriseCustomerUsersResponse;
+  try {
+    const {
```

Comment:
[inform] Existing logic is the same, just moved under this `try` block, and ensures the return in the `catch` block returns the same object keys, just empty.